de/deploy/README.md

```diff
@@ -191,11 +191,11 @@ Nun kannst auch einen Blick auf deinen Code auf PythonAnywhere werfen mittels `l
     templates  tests.py  urls.py  views.py
     
 
-Du kannst auch über den "Files"-Tab mit dem PythonAnywhere Built-in File Browser navigieren.
+Du kannst auch auf die "Files"-Seite gehen und mit PythonAnywheres eingebautem Datei-Manager navigieren. (Von der "Console"-Seite gelangst über das Menü in der rechten oberen Ecke zu anderen PythonAnywhere-Seiten. Sobald du auf einer dieser Seiten bist, sind oben links zu den anderen dieser Seiten.)
 
```

Comment:
Should it be "Links" in meaning of  "hyperlinks", or is it "links" in meaning of "left" and sth else is missing here. Can you please help, @das-g ?

Reply:
In the English original, it says:
> Once you're on one of the pages, there are links to the other ones near the top.)

Thus, it should be the **noun "Links" (as in "Hyperlinks", engl. "links")**, not the position "links" ("left"). I've now capitalized the word on Crowdin.

Reply:
Do you have any suggestions on how to better phrase my "zu den anderen dieser Seiten" (which is still rather confusing)? Or should we completely rephrase that sentence? (The "oben Links" (links at the top) vs. "oben links" (at the top left) confusion might even occur with the fixed spelling.)

Reply:
I see, thank you @das-g. I think that's why I got stuck here, due to "zu den anderen dieser Seiten" (thought they might be displayed on the left side). I checked this now on pythonanywhere, the menu is collapsed if you are in a working mode (like using the terminal or editing a file) and horizontal if you are just browsing. A bit confusing UI in that sense. - Maybe "Sobald du auf einer dieser Seiten bist, findest du die Links zu den anderen Seiten im horizontalen Menü". Or we don't mention it at all?

Reply:
The "Sobald ..., findest du ..." construct is good! Though I wouldn't write "im horizontalen Menü" as that doesn't say where to find that. (Could be at the bottom of the page.) So maybe

> Sobald du auf einer dieser Seiten bist, findest du die Links zu den anderen Seiten oben über dem Seiteninhalt.

Reply:
I will change it, and I think otherwise we are done for that round.